index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
```

Comment:
How the README should be
![image](https://user-images.githubusercontent.com/84156714/235037792-4d91cdbd-610c-4818-bc60-5d22f9e1443a.png)


Your README 
![image](https://user-images.githubusercontent.com/84156714/235037627-b8b5eddc-1549-42ee-b7a3-8902fb259382.png)